packages/types/src/currency-types.ts

```diff
@@ -6,6 +6,7 @@ export type EvmChainName =
   | 'arbitrum-one'
   | 'arbitrum-rinkeby'
   | 'avalanche'
+  | 'base'
```

Comment:
'base' is already on line 33. I recommend leaving this change and removing it from line 33 to maintain alphabetical order.